modules/rosa-policies-to-roles.adoc

```diff
@@ -0,0 +1,92 @@
+
+// Module included in the following assemblies:
+//
+// * rosa_cli/rosa-manage-objects-cli.adoc
+:_mod-docs-content-type: REFERENCE
+[id="rosa-policies-to-roles_{context}"]
+= Manage policies and roles
+
+This section describes the `attach` and `detach` commands. These commands are used to attach policies to roles and detach policies from roles.
+
+[IMPORTANT]
+====
+The `attach` and `detach` commands only apply to Red Hat-managed roles.
+====
+
+[id="rosa-attach-policies_{context}"]
+== Attach policies to roles
+
+Attach policies to roles.
+
+.Syntax
+[source,terminal]
+----
+$ rosa attach policy --policy-arns <policy_arn_1>,<policy_arn_2> --role-arn <role_arn> --mode auto|manual
+----
+
+.Arguments
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--policy-arns
+|Required: The Amazon Resource Names (arns) of the policies that you want to attach to the selected role.
+|--role-arn
+|Required: The arn of the role that you want to attach the selected policies to.
+|--mode
+|Required: In auto mode, changes will be explicitly applied through the user's client. In manual mode, the commands necessary to attach policies will be outputted.
+|===
+
+.Optional arguments inherited from parent commands
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--help
+|Shows help for this command.
+
+|--debug
+|Enables debug mode.
+
+|--yes
+|Automatically answers `yes` to confirm the operation.
+|===
+
+[id="rosa-detach-policies_{context}"]
+== Detach policies from roles
+
+Detach policies from roles.
+
+.Syntax
+[source,terminal]
+----
+$ rosa detach policy --policy-arns <policy_arn_1>,<policy_arn_2> --role-arn <role_arn> --mode auto|manual
+----
+
+.Arguments
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--policy-arns
+|Required: The Amazon Resource Names (arns) of the policies that you want to detach from the selected role.
+|--role-arn
```

Comment:
It should be 'role-name' and its description, not '--role-arn'

Reply:
This command has been removed from this PR entirely (will be featured in a different XCMSTRAT) so the comment is no longer relevant to this PR. Thx